README.md

```diff
@@ -93,18 +93,32 @@ Your contributions are always welcome!
 
 #### Cross Platform
 - [aQuran](https://github.com/forabi/aQuran) : A mobile-optimized, cross-platform Quran app built with web technologies. http://forabi.github.io/aQuran
+- [Hadith Encyclopedia](https://github.com/fekracomputers/HadithEncyclopediaMobile) cross platform hadith app with coloring for narrators
 
 #### Android
-
-  - [Quran Android] (https://github.com/quran/quran_android) : a quran reading application for android http://android.quran.com
-  - [HisnulMuslim](https://github.com/khalid-hussain/HisnulMuslim) : an Android app which include all the dua from Hisnul Muslim.
-  - [MatnAjrumiyyah](https://github.com/khalid-hussain/MatnAjrumiyyah) : an Android app for Matn Al-Ajurumiyyah.
-  - [Kids Quran Android](https://github.com/abodehq/Kids-Quran-Android-v1) : an interactive app for motivating children to learn Quran
-  - [Athkar Kids](https://github.com/abodehq/Athkar-Kids) : an app to encourage children and help them learn and say different athkar(Azkar)
-  - [Hijri Calendar](https://github.com/abodehq/Hijri-Calendar) : Hijri Calendar [app](https://play.google.com/store/apps/details?id=com.mos7af.hijri) for android
-  - [Huson Almuslim Android V2](https://github.com/abodehq/Huson-Almuslim-Android-v2) : an Android app which include all the dua from Huson Almuslim.
-  - [Islamic Stremaer Android](https://github.com/hammady/Islamic-Streamer-Android) : Stream Islamic audio content (Quran, lessons, videos, ...) on your Android phone
+##### Quran
+  - [Quran Android](https://github.com/quran/quran_android) : a quran reading application for android http://android.quran.com
   - [alfanous-android](https://github.com/AhmedSalem/alfanous-android) : mobile app client for [alfanous](http://www.alfanous.org)
+  - [Quran for Android by Fekra computers](https://github.com/fekracomputers/QuranAndroid) :  free Quran android application with audio recitation
```

Comment:
I didn't work personally on that project,however I don't think currently there is any distinguishing features this project was re-implementation for Quran android.